client/pkg/verify/verify.go

```diff
@@ -64,17 +66,29 @@ func DisableVerifications() func() {
 	}
 }
 
+// Assert will panic with a given formatted message if the given condition is false.
```

Comment:
Revert the change in order of functions so there is no changes to Assert function.